.github/workflows/e2e.yml

```diff
@@ -31,17 +31,38 @@ jobs:
             echo "is_pr=false" >> "$GITHUB_OUTPUT"
           fi
 
+      - name: Install gh CLI
+        run: |
+          (type -p wget >/dev/null || (sudo apt update && sudo apt-get install wget -y)) \
+          && sudo mkdir -p -m 755 /etc/apt/keyrings \
```

Comment:
Do we need all the `&&`s in here to have a single command? I think we can use multiple lines since the step will stop when any command fails.

This should also probably be 2 steps: Setup apt, install gh cli.